code/defines/procs/dbcore.dm

```diff
@@ -131,10 +131,9 @@ DBQuery/proc/Execute(var/list/argument_list = null, var/pass_not_found = 0, sql_
 		error("SQL Error: '[error]'")
 		// This is hacky and should probably be changed
 		if (error == "MySQL server has gone away")
-			log_and_message_admins("is attempting to reconnect the server to MySQL. (Connection Failure)")
+			log_game("MySQL connection drop detected, attempting to reconnect.")
+			message_admins("MySQL connection drop detected, attempting to reconnect.")
 			dbcon.Reconnect()

```

Comment:
Actually. This should be db_connect, referencing the source variable. Just in case we use two or more DB entities.